src/styles/blocs/card.scss

```diff
@@ -0,0 +1,112 @@
+.catalog__card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding-inline: 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+
+  &-img {
+    height: 134px;
+    width: 160px;
+    display: block;
+    margin: 0 auto 40px;
+    flex-shrink: 0;
+  }
+
+  &-title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color:#060B35;
+    margin: 0;
```

Comment:
reset margin for `h2` and ``p tags to avoid resetting this value each time at class selectors. check cases below as well